projects/uitpas/docs/errors.md

```diff
@@ -0,0 +1,341 @@
+
+## passholder-not-found
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/passholder-not-found`
+* **Title**: `Passholder not found`
+* **Status**: `404`
+
+The request you are trying to perform cannot complete because the provided passholder cannot be found.
+
+## pass-not-found
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/pass-not-found`
+* **Title**: `Pass not found`
+* **Status**: `404`
+
+The request you are trying to perform cannot complete because the provided pass cannot be found.
+
+## card-blocked
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/card-blocked`
+* **Title**: `Card blocked`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the provided card is blocked.
+
+## event-not-found
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/event-not-found`
+* **Title**: `Event not found`
+* **Status**: `404`
+
+The request you are trying to perform cannot complete because the provided event cannot be found.
+
+## cardsystem-not-found
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/cardsystem-not-found`
+* **Title**: `Card system not found`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the provided card system cannot be found.
+
+## organizer-not-found
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/organizer-not-found`
+* **Title**: `Organizer not found`
+* **Status**: `404`
+
+The request you are trying to perform cannot complete because the provided organizer cannot be found.
+
+## invalid-uitpas-number
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/invalid-uitpas-number`
+* **Title**: `Invalid uitpas number`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the provided uitpas number is invalid.
+
+## invalid-insz-number
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/invalid-insz-number`
+* **Title**: `Invalid insz number`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the provided insz number is invalid.
+
+## invalid-card-system
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/invalid-card-system`
+* **Title**: `Invalid card system`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the provided card system is not applicable in this context.
+
+## disributionkey-not-found
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/disributionkey-not-found`
+* **Title**: `Distribution key not found`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the provided distribution key cannot be found.
+
+## invalid-distributionkey
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/invalid-distributionkey`
+* **Title**: `Invalid distribution key`
+* **Status**: `500`
+
+The request you are trying to perform cannot complete because the no valid distribution key can be found.
```

Comment:
```suggestion
The request you are trying to perform cannot complete because no valid distribution key can be found.
```

Reply:
@erwin1 I think this typo is still present after your recent changes. Not sure if you can still automatically apply the suggestion after the new sorting though.